stackslib/src/chainstate/stacks/boot/signers.clar

```diff
@@ -0,0 +1,22 @@
+(define-data-var last-set-cycle uint u0)
+(define-data-var stackerdb-signer-slots (list 4000 { signer: principal, num-slots: uint }) (list))
+
+(define-private (stackerdb-set-signer-slots 
+                   (signer-slots (list 4000 { signer: principal, num-slots: uint }))
+                   (reward-cycle uint))
+	(begin
+		(print signer-slots)
+        (var-set last-set-cycle reward-cycle)
+		(ok (var-set stackerdb-signer-slots signer-slots))))
+
+(define-read-only (stackerdb-get-signer-slots)
+	(ok (var-get stackerdb-signer-slots)))
+
+(define-read-only (stackerdb-get-config)
+	(ok
+		{ chunk-size: u4096,
```

Comment:
`chunk-size` should be about 1 MB (cc @xoloki -- how big can DKG / signing round messages be?  We can go up to 16 MB).

Reply:
Updated in 5510b1769 (set to 2MB) in the upstream branch.